package.json

```diff
@@ -50,12 +49,17 @@
     "which": "^5.0.0"
   },
   "devDependencies": {
+    "@types/busboy": "~1.5.4",
+    "@types/connect": "~3.4.38",
+    "@types/cookie-signature": "~1.1.2",
     "@types/cross-spawn": "^6.0.6",
     "@types/fs-extra": "^11.0.4",
+    "@types/jsesc": "~3.0.3",
     "@types/make-fetch-happen": "^10.0.4",
     "@types/micromatch": "^4.0.9",
     "@types/node": "^22.10.0",
     "@types/serve-static": "^1.15.7",
+    "@types/which": "~3.0.4",
```

Comment:
TS was complaining about the lack of types for which, so I ran typesync.
Note that it uses a tilde, as DefinitelyTyped keeps majors and minors in sync with the upstream. Now, there aren't super recent types for `which`, so it's a bit pointless, but it just did it automatically.